src/components/Calculator/ScoreTable.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import getCourseScore from '../../utils/getCourseScore'
+
+import coursesArray from '../../courses.json'
+
+const ScoreTable = ({ subjects, onToggleClick }) => (
+  <div>
+    <br />
+    <br />
+    <table className='table table-bordered score-table'>
+      <tbody>
+        <tr>
+          <th scope='row'>
+            Wydział
+          </th>
+          <th scope='row'>
+            Kierunek
+          </th>
+          {/* <th scope='row'>
+            Przedmioty
+          </th> */}
+          <th scope='row'>
+            Wynik
+          </th>
+        </tr>
+        {coursesArray.map((course, i, coursesArray2) =>
+          <tr key={i}>
+            <td>
+              <strong>
+                {(i !== 0 && coursesArray2[i - 1].faculty === course.faculty) ? null : course.faculty }
```

Comment:
The JSON converter I found couldn't realise that idea, so I didn't want to do it manually. But it was worth, so now I did. Do you have some good tools for this kind of stuff?